app/helpers/products_helper.rb

```diff
@@ -47,13 +47,15 @@ def set_sorting(test, test_status)
 
     case test_status
     when 'passing'
-      return 5
+      return 6
```

Comment:
I think it might make even more sense to make an array of the possible statuses and then set the sort order based off indexes in that array. Easier to change around the order or add a status that way